ee/desktop/menu/menu.go

```diff
@@ -14,8 +14,7 @@ import (
 type menuIcon string
 
 const (
-	KolideDesktopIcon      = "kolide-desktop"
-	KolideDebugDesktopIcon = "kolide-debug-desktop"
```

Comment:
Looks like removing this const is making the menu parse tests unhappy, may need to update those tests too?